src/debug.c

```diff
@@ -1522,7 +1522,11 @@ void logRegisters(ucontext_t *uc) {
  * Close it with closeDirectLogFiledes(). */
 int openDirectLogFiledes(void) {
     int log_to_stdout = server.logfile[0] == '\0';
-    int fd = log_to_stdout ?
+    int fd;
+
+    if (strcasecmp(server.logfile, "none") == 0) return -1;
+
+    fd = log_to_stdout ?
```

Comment:
No need to init the `fd` variable on two separate lines, is there?

```suggestion
    if (strcasecmp(server.logfile, "none") == 0) return -1;

    int fd = log_to_stdout ?
```

Reply:
The fd variable is not initialized twice, it's declared once and initialized afterwards. Merging the declaration and first assignment would mean the code mixes declarations in between statements which does not seem like what the coding style of this project is doing.

Reply:
Fair enough! Consistency is king.

Reply:
I guess that depends on your perms, so only author and project owners see those. I do have the button, so I'll mark as resolved. :)